modules/httpd/manifests/init.pp

```diff
@@ -5,29 +5,24 @@
 # @example
 #   include httpd
 
-class httpd {
+class httpd (
+  Boolean $welcome_page = false,
+  ){
   package {'httpd':
     ensure => installed,
-    before => File['/etc/httpd/edit_httpdconf.sh'],
   }
 
-  file { '/etc/httpd/edit_httpdconf.sh':
-    ensure  => file,
-    owner   => 'root',
-    group   => 'root',
-    mode    => '0755',
-    content => file( 'httpd/edit_httpdconf.sh'),
-    notify  => Exec['edit_httpd.conf'],
+  service {'httpd':
+    ensure => running,
+    enable => true,
   }
 
-  exec {'edit_httpd.conf':
-    command  => '/etc/httpd/edit_httpdconf.sh',
-    path     => '/bin:/sbin:/usr/bin:/usr/sbin'
+  if $welcome_page {
```

Comment:
What should this code do?

Reply:
It checks for Apache default test page, and by default deletes it.